generators/generator-base-private.js

```diff
@@ -54,6 +55,7 @@ module.exports = class extends Generator {
         this.env.options.appPath = this.config.get('appPath') || CLIENT_MAIN_SRC_DIR;
         // expose lodash to templates
         this._ = _;
+        this.createConfigFromNewConfFile();
```

Comment:
@MathieuAA why is this needed, could you please explain?

Reply:
you need this only for import-jdl right? i'm worried about the blueprints, as I dont know what side effect this would add to all generators

Reply:
I think I caught your meaning, I can try to make it only possible for the import-jdl subgen, now that I understand how the Storage works.